server/src/parseResponse.ts

```diff
@@ -0,0 +1,41 @@
+import { ResponseData } from "./utils/types";
+
+// Calculate display time based on reading speed
+const calculateDisplayTime = (text: string, wordsPerMinute: number = 200): number => {
+const words = text.split(' ').length;
+const minutesToRead = words / wordsPerMinute;
+const secondsToRead = minutesToRead * 60;
```

Comment:
actually one last thing, it may be better to use milliseconds.
probably a little easier for the UI since a lot of style/timeout stuff is in milliseconds

maybe we can use `200 * 60 * 1000` as the default value for `wordsPerMinute` and then no need to use the extra lines of code to convert to seconds?

Reply:
one last last note, for all our magic numbers, like `wordsPerMinute` and `baseDisplayTime`, can we move them to a file in the `utils` folder called `constants.ts` and declare them there instead?

```
// in milliseconds
export const WORDS_PER_MS = 12_000_000;
export const BASE_DISPLAY_TIME = 120_000;
```

Reply:
will do!